content/en/code_analysis/software_composition_analysis/setup.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Setting Up Software Composition Analysis
+kind: documentation
+description: Learn how to set up Software Composition Analysis to scan your imported open-source libraries for known security vulnerabilities before you ship to production.
+is_beta: true
+further_reading:
+- link: "https://www.datadoghq.com/blog/iast-datadog-application-vulnerability-management/"
+  tag: "Blog"
+  text: "Enhance application security in production with Datadog Application Vulnerability Management"
+- link: "/getting_started/application_security/software_composition_analysis"
+  tag: "Documentation"
+  text: "Getting Started with Software Composition Analysis"
+- link: "/security/application_security/software_composition_analysis/"
+  tag: "Documentation"
+  text: "Learn about Software Composition Analysis"
+- link: "/integrations/guide/source-code-integration/"
+  tag: "Documentation"
+  text: "Learn about the Source Code Integration"
+- link: "/code_analysis/static_analysis/"
+  tag: "Documentation"
+  text: "Learn about Static Analysis"
+---
+
+{{% site-region region="gov" %}}
+<div class="alert alert-danger">
+    Code Analysis is not available for the {{< region-param key="dd_site_name" >}} site.
+</div>
+{{% /site-region %}}
+
+{{< callout url="#" btn_hidden="true" header="Try the Beta!" >}}
+Code Analysis is in public beta.
+{{< /callout >}}
+
+## Overview
+
+To use Datadog Software Composition Analysis (SCA) in CI, you only need to set it up in its own dedicated job. 
+
+## Enable Software Composition Analysis
+
+When setting up [Code Analysis][2] on your project, select **Enable Software Composition Analysis**.
```

Comment:
```suggestion
When setting up [Code Analysis][2] on your project, select **Enable Software Composition Analysis**.
```
Is there a place in-app we could link to here as well? Such as "Navigate to...., select .." ?